tests/function-controller/hack/k8s_framework_create.patch

```diff
@@ -0,0 +1,71 @@
+--- vendor/k8s.io/kubernetes/test/e2e/framework/create.go.orig	2019-09-30 13:06:49.391547627 +0200
```

Comment:
Is this something that we can get upstream?

Reply:
The framework ships with a super convenient function that can create API objects from a list of YAML manifests while automatically using the correct namespace, and then returns a function to clean them up in a single call. Mindblowing ⭐️

Unfortunately that function does not support all object types 😢 Instead of rewriting the whole thing, I figured I could simply inject those types via a patch, since we only need _2_.

Pushing extra types upstream is the better approach, but then we'd have to update the framework to use k8s' `master`, which we should avoid.